scripts/start-logging.ps1

```
@@ -0,0 +1,75 @@
+        $pattern = "(QuestHook|modloader|"
+    }
+    $pattern += "AndroidRuntime|CRASH)"
+    $command += " | Select-String -pattern `"$pattern`""
+}
+
+if ($file -eq $true) {
+    $command += " | Out-File -FilePath $PSScriptRoot\..\log.log"
+}
+
+Write-Output "Logging using Command `"$command`""
+Invoke-Expression $command
```

PSScriptAnalyzer (1.25.0)

[warning] Invoke-Expression is used. Please remove Invoke-Expression from script and find other options instead.

(PSAvoidUsingInvokeExpression)